Automattic/sensei
# Add mobile support for the new email templates

Depends on #6666 
Fix #6640 

## Proposed Changes
* Extract the email template styles to a separate file
* Add a media query to support email clients. 
* Update the Teacher > Course Started pattern to support the mobile email clients.

## Tech Notes
* I am using a lot of ugly `!important` instructions because it is required by Emogrifier to render the media styles and also recommended by Gmail. 
* Gmail doesn't support the traditional `@media only screen and (max-width: `. It only supports `@media (max-width:` (without the only screen....).
* I extracted the template CSS to a new file because emogrifier was overwriting them they are processed together with the regular styles. 

## Testing instructions
* Use a Jurassic env + smtp plugin + Gmail configuration to send real e-mails.
* Assing a student to a course
* Check in a mobile device. 
* Don't use mail trap or any email capture plugin to test the HTML rendering, the results are significantly different from the email clients.

 
### Mobile using dark mode
![mobile-dark](https://user-images.githubusercontent.com/38718/226745948-3ae83465-aac4-439f-b9e2-493425f3c01b.jpeg)

![normal-mode](https://user-images.githubusercontent.com/38718/226746284-80f4905a-6027-481d-aa4d-d60c7a204eb0.jpeg)

## Pre-Merge Checklist
